c-api/libvmexeccapi.h

```diff
@@ -43,11 +43,7 @@ typedef struct {
   void (*get_owner_address_func_ptr)(void *context, int32_t result_offset);
   int32_t (*is_smart_contract_func_ptr)(void *context, int32_t address_offset);
   void (*signal_error_func_ptr)(void *context, int32_t message_offset, int32_t message_length);
-  void (*get_external_balance_func_ptr)(void *context, int32_t address_offset, int32_t result_offset);
-  int32_t (*get_block_hash_func_ptr)(void *context, int64_t nonce, int32_t result_offset);
```

Comment:
Don’t think this function should be removed…. how do you come with this? Block hash is a feature common used in several contracts